Update callout about Pro routing (#8841)
magento/devdocs: src/cloud/project/project-multi-sites.md

Routes define how to process incoming URLs. Multiple stores with unique domains requires you to define each domain in the `routes.yaml` file. The way you configure routes depends on how you want your site to operate.

{:.bs-callout-info}

> For Pro, if your routes are not getting configured, you must create a [Support ticket]({{ site.baseurl }}/cloud/trouble/trouble.html) - please include "enable self-service routes" in your request.

Reviewer: Please is nice, but we don't use it.

```suggestion
For Pro, if your route configurations are not setting properly, you can create a [Support ticket]({{ site.baseurl }}/cloud/trouble/trouble.html) and include "enable self-service routes" in your request.
```
